### DISCUSSION TITLE
Verified / Verified Blue

### DISCUSSION THREAD

**Discussion by killabit** [robertodoering/harpy#840] [General]
Seems that Harpy only displays Vetiford badges on iPhone and not the Verified Blue badges.

Anyone know if they are called differently? I haven't been able to find any info about it.

**Comment by robertodoering** [comment]
The new verified badges aren't returned in responses of their api, so we can't show them.

**Reply by dnestico** [reply]
Whenever they do add them to the API you should let us customize the Twitter Blue verified checkmark so I can put a 🤡 for people who paid for that lol

**Reply by killabit** [reply]
LOL
